ui/app/components/oidc-auth-buttons/index.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  AuthMethod,
+  GetAuthMethodRequest,
+  GetOIDCAuthURLRequest,
+  ListOIDCAuthMethodsResponse,
+  Ref,
+} from 'waypoint-pb';
+
+import ApiService from 'waypoint/services/api';
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import { tracked } from '@glimmer/tracking';
+
+interface OIDCAuthButtonsArgs {
+  model: ListOIDCAuthMethodsResponse.AsObject;
+}
+
+export default class OIDCAuthButtonsComponent extends Component<OIDCAuthButtonsArgs> {
+  @tracked model!: ListOIDCAuthMethodsResponse.AsObject;
```

Comment:
Is this still in use?

Reply:
not used here but used in the template. I'll remove the `@tracked` decorator.

Reply:
I see `@model` referenced in the template but not `this.model`. Not a big deal, either way.